src/common/Header.tsx

```diff
@@ -0,0 +1,88 @@
+import styled from 'styled-components';
+import { IcLogo } from '../assets';
+import { DATA } from '../constants/Header/HeaderConst';
+import { HeaderProps } from '../types/Header/HeaderType';
+
+const Header = ({
+  isLogin,
+  nickname,
+  clickedCategory,
+  handleClickCategory,
+}: HeaderProps) => {
+  return (
+    <HeaderContainer>
+      <LogoContainer>
+        <IcLogo />
+      </LogoContainer>
+      <NavBarContainer>
+        {DATA.map((v) => {
+          return (
+            <NavBar key={v.text}>
+              {isLogin && clickedCategory === v.text && (
+                <IconContainer>{v.icon}</IconContainer>
+              )}
+              <Text
+                onClick={(e) => handleClickCategory(e)}
+                $isClickedCategory={isLogin && clickedCategory === v.text}
+              >
+                {v.text}
+              </Text>
+            </NavBar>
+          );
+        })}
+      </NavBarContainer>
```

Comment:
P2: nav 안에 ul이나 ol없이 li가 바로 들어가는 형태는 좋은 패턴이 아니라고 합니당
HTML5에서 nav는 링크의 집합을 정의하는데 사용하는 것이 일반적이기 때문에, li를 ul이나 ol로 감싸서 그룹화를 한 뒤
nav 안에 넣어주면 좋을 것 같습니다 !
```suggestion
      <NavBarContainer>
        <ul>
          {DATA.map((v) => {
            return (
              <NavBar key={v.text}>
                {isLogin && clickedCategory === v.text && (
                  <IconContainer>{v.icon}</IconContainer>
                )}
                <Text
                  onClick={(e) => handleClickCategory(e)}
                  $isClickedCategory={isLogin && clickedCategory === v.text}
                >
                  {v.text}
                </Text>
              </NavBar>
            );
          })}
        </ul>
      </NavBarContainer>
```

Reply:
확인하구 수정 하였습니다 감사합니다~!